Add nav-bar to xnd.io

andersy005 opened the pull request:
This PR 
- Adds a navigation bar/menu to xnd.io with links to documentation pages of xnd-related projects.
- Introduces some refactoring to the repo by moving images to their own directory.

Here is a preview of the new page (https://andersy005.github.io/website/)

andersy005 committed:
move images to their own directory

andersy005 committed:
disable CNAME for debug purposes

andersy005 committed:
add navigation menu

andersy005 committed:
update navigation bar

andersy005 committed:
add links to other projects docs

andersy005 committed:
enable CNAME

xmnlab commented:
It seems good for me.

it just needs a link for install and tutorial. maybe would be good to remove this options for now.

andersy005 committed:
remove tutorial and install sections

xmnlab commented:
thanks @andersy005 !

I am just thinking about `Motivation` menu. I think it is not necessary ... it just send the user a little bit bellow.

I think for now just the "Documentation" menu is enough. In the next task probably we will add new menus there :)

LGTM .. just need this little change.

waiting for @saulshanabrook feedback.

saulshanabrook commented:
Thanks for tackling this! I like the overall approach. I have a few comments:

* Could everything under "Documentation" be at the top level? Also, there is now some repetition from the "Packages" sections and the header. Should everything in the header have a section under the "Packages"? If so, maybe the top header just scrolls you to the right section?
* The footer seems to be floating now:

<img width="1538" alt="screen shot 2018-07-25 at 5 43 41 pm" src="https://user-images.githubusercontent.com/1186124/43229302-4ea4c5c2-9032-11e8-8b50-002b1dfdb429.png">

* Do we need the custom CSS and jquery? I notice the fonts are a bit lighter now. Are there other goals to add the CSS besides that? I think bootstrap can support navbars without jquery: https://getbootstrap.com/docs/4.1/components/navbar/, but I could be wrong. 

* When it's under a certain width, the hamburger colour seems a bit dark:
<img width="957" alt="screen shot 2018-07-25 at 5 48 12 pm" src="https://user-images.githubusercontent.com/1186124/43229510-f9d13886-9032-11e8-9c59-33d6084e8957.png">

* I am not in favor of the redundant icons, both in the big logo and the top left. I see now that is what dask does. They don't have any letters in the top left icon. It might look better that way, if we coudl get an XND one without the "XND" text. The icon in the top left is also too blurry. I can look for high res versions of them if you need, someone sent them to me at one point.

* It might also be nice to make the navbar a different tone than the header. For example, this bootstrap example doesn't look bad: https://getbootstrap.com/docs/4.1/examples/jumbotron/

saulshanabrook reviewed (changes requested):
Added some comments.

andersy005 committed:
remove unncessary  css and  js files

andersy005 committed:
update background color

andersy005 commented:
Thank you for the feedback, @saulshoanabrook! 
- I moved everything under "Documentation" to be at the top level and avoided the redundancy by making sure that the header just scrolls down to the right section. 
- I agree with you on the fact that extra css and js are not needed. I have removed them and I am going to take full advantage of Bootstrap.
 - Do you have any xnd logo without text that I could use for `navbar brand` section? Let me know what other alternative(s) we can use here.

saulshanabrook commented:
@andersy005 That was quick, thanks!

Here are the two logo files Emma originally sent me:

![](https://slack-files.com/T8XM7KN5B-FAJEVDS1H-ba95a3074a)
![](https://slack-files.com/T8XM7KN5B-FAJHUKWP4-0ddec25dbb)

I cropped/shrunk each for the site.

xmnlab reviewed (changes requested):
Thanks @andersy005 !

just some comments.

1. I think you could remove `Navbar` from the navbar ... you can just leave this empty.
2. xndtools need a link, we don't have any text for xndtools on the xnd.io page, so you can just link to https://github.com/plures/xndtools

I think after that we can merge :+1:

andersy005 committed:
Update navbar-brand section and add xnd_tools link

andersy005 commented:
@saulshanabrook, 

 I just pushed new changes. Thanks for the logo files. I cropped one of the two and used it for navbar-brand section. 

Also, I found out that the navigation bar/menu would not be responsive without Jquery. So, I have added it too. 

@xmnlab, 

For now, xndtools section links to the github repo. Feel free to update it once you add xndtools docs to readthedocs xnd-all webpage.

andersy005 committed:
change navbar color to white

xmnlab reviewed (approved):
LGTM! thank you so much @andersy005 !

@saulshanabrook do you want to do the last check on the webpage before merging?  https://andersy005.github.io/website/

saulshanabrook commented:
Thanks Anderson, this looks great! I just merged, but the. Realized you still have wip in the title. If there are other lingering changes, I guess they can go in a new pr
